static/app/views/replays/detail/focusArea.tsx

```diff
@@ -102,4 +107,19 @@ function FocusArea({}: Props) {
   }
 }
 
+const StyledTrace = styled(Trace)`
+  margin-top: 0;
+  height: 100%;
+
+  ${TracePanel} {
+    height: 100%;
+    overflow: auto;
+  }
+  ${TraceViewHeaderContainer} {
+    position: sticky;
+    top: 0;
+    z-index: 1;
+  }
+`;
```

Comment:
I know @markstory wanted to stop using styled components as selectors, but not sure the best way to do it without it.

Explicit props for each component that we need to restyle and using `css` from the emotion library? e.g. tracePanelCss={css`height: 100%;overflow:auto;`}

Reply:
I can agree with mark on that. I'm souring on `styled(Trace)` to override styles -> imo it means the base component is doing too much and there's no api for callers to override things. Changing the base styles is risky. 
_chanting:_ `style-d` for native nodes on-ly!

Having explicit css props for the child nodes could blow up the interface for `<Trace>` though... i'll take a look since we're pretty good about not doing this in replays so far.

Reply:
Much better now, but need help testing one more spot 🥇